html-aam/index.html

```diff
@@ -849,8 +862,30 @@ <h4 id="el-autonomous-custom-element">autonomous custom element</h4>
             </tr>
             <!-- <th><a href="#accessible-name-and-description-computation">Naming Algorithm</a></th> -->
             <tr>
-              <th>Comments</th>
-              <td></td>
+              <th id=custom-element-comments>Comments</th>
+              <td>
+                <p>
+                  Along with the attributes defined in this specification that contribute to modifying an element's <a>minimum role</a>,
+                  the following conditions will also alter a custom element's computed role, if an author has not otherwise specified an
+                  explicit role for the element:
+                </p>
+                <ul>
+                  <li>if the custom element is focusable: expose as <a class="core-mapping" href="#role-map-group">`group`</a> role</li>
+                  <li>if the custom element has an `aria-live` or `aria-owns` attribute: expose as <a class="core-mapping" href="#role-map-generic">`generic`</a> role</li>
+                  <li>if element internals are used to set a global ARIA property: expose as <a class="core-mapping" href="#role-map-generic">`generic`</a> role</li>
+                  <li>if an author specifies an ARIA relation via an element attribute: expose <a class="core-mapping" href="#role-map-generic">`generic`</a> role</li>
+                </ul>
+                <div class="example">
+                  <p>The following is an exmaple of an author specifiying an ARIA relation on a custom element, with another element in the DOM.</p>
```

Comment:
ex-maple, not ex-oak?

Reply:
Seriously, is this example for the previous bullet? Because aria-details won't change it from 'none', unless you do myCustomElement.ariaDetailsElements = [ el1, ... ];

Reply:
ah ok.  thanks.  I didn't realize you were saying that this would only happen if it was set via IDL.

my thought was that this was another use case that didn't make sense to reflect, since the ID that was being referenced would likely be in the light DOM, so it wouldn't work if reflected into the custom element's shadow dom parts